docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/media/video/video_macros.twig

```diff
@@ -28,12 +31,14 @@
           <p>{{ name }}</p>
         </div>
       </div>
-      {% set hasCaption = content.field_caption|field_value %}
       {% if hasCaption and showCaption %}
-        <figcaption class="caption-container">
+        <figcaption class="caption-container" id="{{ videoID }}">
           {{ content.field_caption }}
         </figcaption>
       {% endif %}
     </figure>
+    {% if hasAudioDescribed %}
```

Comment:
This should be in some sort of container... So if it must come after the figure, then you need to be elevating the class to have some sort of block element.